### DISCUSSION TITLE
is Symfony 6.0 in dev statge?

### DISCUSSION THREAD

**Discussion by gopikrish19** [symfony/symfony#44347] [Q&A] [answered]
Hi,
     I create my symfony project with symfony CLI new command, it create the project with 5.4 version, and i can able to get the symfony 6.0 by  this command  "composer create-project symfony/skeleton:"6.0.x@dev" test"

**Comment by ahundiak** [comment, marked as answer]
You probably just happened to catch Symfony in the process of releasing.  symfony new now installs 6.0 while symfony new --version=lts gives you 5.4.0.

Probably also want to make sure you run composer self:update as well as symfony self:update.  And of course have at least PHP 8.0.2 installed.
